.buildkite/validate-podspec.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
+
+echo "--- :microscope: Validate Podspec"
+xcrun simctl list >> /dev/null # For some reason this fixes a failure in `lib lint`
```

Comment:
😖 ❓ 

Do we happen to have a GitHub issue to link here? Just in case it's a problem upstream (CocoaPods?) and it'll be fixed eventually?

Reply:
This _may_ be something to do with the VM? I've opened an issue in https://github.com/Automattic/buildkite-ci/issues/7 to track it!